WORKSPACE

```diff
@@ -154,16 +154,16 @@ load("@com_github_bazelbuild_buildtools//buildifier:deps.bzl", "buildifier_depen
 
 buildifier_dependencies()
 
-http_archive(
-    name = "com_github_prysmaticlabs_go_ssz",
-    sha256 = "f6fd5d623a988337810b956ddaf612dce771d9d0f9256934c8f4b1379f1cb2f6",
-    strip_prefix = "go-ssz-2e84733edbac32aca6d47feafc4441e43b10047f",
-    url = "https://github.com/prysmaticlabs/go-ssz/archive/2e84733edbac32aca6d47feafc4441e43b10047f.tar.gz",
-)
+# http_archive(
+#     name = "com_github_prysmaticlabs_go_ssz",
+#     sha256 = "f6fd5d623a988337810b956ddaf612dce771d9d0f9256934c8f4b1379f1cb2f6",
+#     strip_prefix = "go-ssz-2e84733edbac32aca6d47feafc4441e43b10047f",
+#     url = "https://github.com/prysmaticlabs/go-ssz/archive/2e84733edbac32aca6d47feafc4441e43b10047f.tar.gz",
+# )
```

Comment:
This can be a go_repository, but it needs to call `go_ssz_dependencies` too.

Please remove commented code.

Reply:
ok

Reply:
done.